spec/requests/api/v1/posts/update_spec.rb

```diff
@@ -0,0 +1,34 @@
+RSpec.describe Api::V1::PostsController, type: :request do
+  let(:user) { FactoryBot.create(:user, email: 'george@mail.com') }
+  let(:admin) { FactoryBot.create(:user, admin: true, email: 'noel@craft.com') }
+  let(:post) { FactoryBot.create(:post) }
+  let(:credentials) { user.create_new_auth_token }
+  let(:admin_credentials) { admin.create_new_auth_token }
+  let(:headers) { { HTTP_ACCEPT: "application/json" }.merge!(credentials) }
+  let(:admin_headers) { { HTTP_ACCEPT: "application/json" }.merge!(admin_credentials) }
+
+  describe "PATCH /api/v1/posts/id" do
+
+    it "updates status if user is admin" do
+      patch "/api/v1/posts/"+"#{post.id}", params: {
```

Comment:
Why the plus sign?

Reply:
That's true. Removed from this and all other request specs that was used.